__tests__/articles.test.js

```diff
@@ -79,4 +79,73 @@ describe('GET /api/articles', () => {
             });
         });
     });
-});
\ No newline at end of file
+});
+
+describe('PATCH /api/articles/:article_id', () => {
+    test('200: responds with the updated article', () => {
```

Comment:
Just be a bit more descriptive with your test name. This could say "responds with the updated article when votes are incremented" 👍